drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-backend/src/test/java/org/drools/workbench/screens/guided/rule/backend/server/GuidedRuleEditorServiceImplTest.java

```diff
@@ -63,27 +73,30 @@
     @Mock
     private IOService ioService;
 
+    @Mock
+    private DSLSentence dslSentence;
+
     @InjectMocks
     GuidedRuleEditorServiceImpl service = new GuidedRuleEditorServiceImpl(sessionInfo,
                                                                           mock(Instance.class));
 
     @Test
-    public void checkConstructContentPopulateProjectCollectionTypes() {
+    public void checkConstructContentPopulateProjectCollectionTypesAndDSLSentences() throws Exception {
```

Comment:
@triceo I was trying to cover this service with integration test, that would not mock `PackageDataModelOracle` data, however I gave up after like 8 hours of constructing setup for the `GuidedRuleEditorServiceImpl` class. There was like 20 complex constructors called before the test and setup was still failing. Moving to another issue rather than spend more hours here and not be sure if I will succeed.